scripts/aws/common/deploy.py

```diff
@@ -60,7 +60,12 @@ def deploy_api(client, rest_api_id, swagger_filename, stage_name, stage_variable
 
     swagger = open(swagger_filename, 'r', encoding="utf-8")  # pylint: disable=consider-using-with
 
-    api_response = client.put_rest_api(restApiId=rest_api_id, mode='overwrite', body=swagger.read())
+    api_response = client.put_rest_api(
+        restApiId=rest_api_id,
+        mode='overwrite',
```

Comment:
Need to protect against this overwrite operation clobbering existing API object parameters unrelated to this media type

Reply:
We shouldn't add any protection, since the mode is already 'override' in the existing code and we are not introducing anything problematic. In fact, I think the idea is that the repo is the source of truth. According to SRE nothing should be manually set anyway. So override seems correct. If we merge this config together with some settings on AWS, that could lead to mistakes or conflicts. We should leave this on 'override' so that this is the source of truth and anything that's only set on AWS manually, and may introduce some unexpected conflict, gets reset on deploy.